reference/sectionC_interpretation.inc

```diff
@@ -62,7 +62,7 @@ Table: Built-in units
 | farad         | :math:`F`         | (metre, -2), (kilogram, -1),     | | :math:`C/V`                      |
 |               |                   | (second, 4), (ampere, 2)         | | :math:`s/{\Omega}`               |
 +---------------+-------------------+----------------------------------+------------------------------------+
-| gram          | :math:`g`         | 0.001 * (kilogram, 1)            |                                    |
+| gram          | :math:`g`         | :math:`0.001\cdot` (kilogram, 1) |                                    |
```

Comment:
Hmm... the rendering is going to look a bit odd. Part of it is going to be in italics. Why not simply have `0.001⋅(kilogram, 1)`?

Reply:
How do you get the dot without :math: ?  Please feel free to change as you'd like to - these tables take *ages* to make tiny adjustments to!

Reply:
Just write verbatim what I wrote above.

Reply:
Where is the dot symbol?? I can copy yours but I want to know how you got it in the first place?

Reply:
OK, done.  I still want to know how to get that dot though!

Reply:
From [here](https://en.wikipedia.org/wiki/International_System_of_Units#Derived_units). It corresponds to the [dot operator](https://en.wikipedia.org/wiki/Multiplication_sign#Unicode_and_HTML_entities).